hack/install.sh

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DEFAULT_BIN_DIR="/usr/local/bin"
+DEFAULT_CLI="karmadactl"
+CLI=${1:-"${DEFAULT_CLI}"}
+BIN_DIR=${2:-"${DEFAULT_BIN_DIR}"}
+GITHUB_REPO="karmada-io/karmada"
+
+# Helper functions for logs
+info() {
+    echo '[INFO] ' "$@"
```

Comment:
I see other scripts in out project, they are indented with two space. Could you modify it?